README.md

```diff
@@ -1009,6 +1009,89 @@ $ civo network remove 74b69006-ea59-46a0-96c4-63f5bfa290e1
 Removed the network cli-demo with ID 74b69006-ea59-46a0-96c4-63f5bfa290e1
 ```
 
+## Load Balancers
+
+Civo Load Balancers are external to custer, but created and managed as part of cluster's service definitions. In terms of Kubernetes, a Civo Load Balancer is a Service object with rules much like other Service objects in Kubernetes, a Civo Load Balancer is a service object with rules much like other Service type objects in Kubernetes, with a key difference that its state is handled by Cloud Controller MAnnager.
+
+#### Load Balancer configuration options 
```

Comment:
Anything below this is not required as mentioned in https://github.com/civo/cli/pull/222#issuecomment-1100649097